platform/pddf/platform-api-pddf-base/sonic_platform_pddf_base/pddfapi.py

```diff
@@ -11,6 +13,7 @@
 bmc_cache = {}
 cache = {}
 SONIC_CFGGEN_PATH = '/usr/local/bin/sonic-cfggen'
+LED_CTRL_LOCK_PATH = '/var/lock/pddf-api-led.lock'
```

Comment:
Not sure why the control path LED_CTRL_LOCK_PATH in PR32128 is different from PR22684 ? Can you confirm if the PR32128 is required for this PR ?